docs/book/src/getting-started.md

```diff
@@ -33,7 +33,7 @@ kubebuilder init --domain=example.com
 Next, we'll create a new API responsible for deploying and managing our Memcached solution. In this instance, we will utilize the [Deploy Image Plugin][deploy-image] to get a comprehensive code implementation for our solution.
 
 ```
-kubebuilder create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha" --make=false
+kubebuilder create api --group cache --version v1alpha1 --kind Memcached --image=memcached:1.4.36-alpine --image-container-command="memcached,-m=64,-o,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha" --make=false
```

Comment:
Are we using cache to generate the example under `testdata/project-v4-with-deploy-image/internal/controller/memcached_controller.go`? Could you please check the shell script that we use? If you want to change it please could you either change the testdata shell and re-run make generate to update the results?